public/css/style.css

```diff
@@ -0,0 +1,83 @@
+.container {
+  margin: 0 auto;
+  border-width: 14px;
+  border-style: solid;
+  border-color: pink;
+  border-radius: 95px;
+  text-align: center;
+  width: 600px;
+  height: 775px;
+  background-color: lightblue;
+  font-family: 'Comfortaa', cursive;
+  color: black;
+}
+
+.secret-word {
+  font-family: 'Inconsolata', monospace;
+  font-size: 40px;
+  font-weight: bold;
+  padding-top: 15px;
+}
+
+.link {
+  border-radius: 12px;
+  background-color: pink;
+  padding-top: 25px;
+  width: 110px;
+  height: 10px;
+  margin-left: auto;
```

Comment:
There is a way to combine all these margin values into one line. Have a look if you can figure it out.